hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -923,15 +923,18 @@ def status(format, type)
         for v in cluster_metrics.getRegionStatesInTransition
           puts(format('    %s', v))
         end
-        master = cluster_metrics.getMasterName
-        puts(format('active master:  %s:%d %d', master.getHostname, master.getPort,
-                    master.getStartcode))
-        puts(format('%d backup masters', cluster_metrics.getBackupMasterNames.size))
-        for server in cluster_metrics.getBackupMasterNames
+        master = cluster_metrics.getMaster
+        unless master.nil?
+          puts(format('active master:  %s:%d %d', master.getHostname, master.getPort, master.getStartcode))
+          for task in cluster_metrics.getMasterTasks
+            puts(format('    %s', task.toString))
+          end
+        end
+        puts(format('%d backup masters', cluster_metrics.getBackupMastersSize))
+        for server in cluster_metrics.getBackupMasters
           puts(format('    %s:%d %d', server.getHostname, server.getPort, server.getStartcode))
         end
-
-        master_coprocs = @admin.getMasterCoprocessorNames.toString
+        master_coprocs = java.util.Arrays.toString(@admin.getMasterCoprocessors)
```

Comment:
This seems additional change, is it for better readability or it's just a mechanical change?